From 8e086773badcd04bc324554402004d2eabbb98b4 Mon Sep 17 00:00:00 2001
From: Pierre Lebleu <pierre.lebleu@pile-engineering.com>
Date: Mon, 4 May 2026 14:45:12 +0200
Subject: [PATCH] Fix: -Wmaybe-uninitialized for image_type in
 gdImageCreateFromTiffCtx

```
src/gd_tiff.c: In function 'gdImageCreateFromTiffCtx':

src/gd_tiff.c:955:43: error: 'image_type' may be used uninitialized [-Werror=maybe-uninitialized]
  955 |                 (image_type == GD_PALETTE || image_type == GD_INDEXED || image_type == GD_GRAY)) {
      |                                           ^

src/gd_tiff.c:819:25: note: 'image_type' was declared here
  819 |         int             image_type;
      |                         ^~~~~~~~~~

'''

Signed-off-by: Pierre Lebleu <pierre.lebleu@pile-engineering.com>
---
 src/gd_tiff.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gd_tiff.c b/src/gd_tiff.c
index 6f50b667f..e88c66a08 100644
--- a/src/gd_tiff.c
+++ b/src/gd_tiff.c
@@ -816,7 +816,7 @@ BGD_DECLARE(gdImagePtr) gdImageCreateFromTiffCtx(gdIOCtx *infile)
 	char	has_alpha, is_bw, is_gray;
 	char	force_rgba = FALSE;
 	char	save_transparent;
-	int		image_type;
+	int		image_type = 0;
 	int   ret;
 	float res_float;
 
